packages/shared/web/src/header/mobile-navigation/mobile-navigation.tsx

```diff
@@ -82,7 +80,7 @@ export function MobileNavigation({ ...rest }: TabsProps) {
   return window.innerWidth <= theme.breakpoints.values.sm ? (
     <>
       <Box sx={{ display: 'flex', justifyContent: 'end' }}>
-        {!wallet?.selectedAddress && isAccountPending && pathname !== '/' && (
+        {!wallet.userWallet?.selectedAddress && pathname !== '/' && (
```

Comment:
Should we keep an isAccountPending flag here still? we can just stub one out instead of just deleting it?